code/modules/antagonists/changeling/powers/chameleon_skin.dm

```diff
@@ -1,25 +1,43 @@
 /datum/action/changeling/chameleon_skin
-	name = "Chameleon Skin"
-	desc = "Our skin pigmentation rapidly changes to suit our current environment. Costs 25 chemicals."
-	helptext = "Allows us to become invisible after a few seconds of standing still. Can be toggled on and off."
+	name = "Darkness Adaptation"
+	desc = "Our skin pigmentation and eyes rapidly changes to suit the darkness. Needs 25 chemicals in-hand to toggle."
+	helptext = "Allows us to darken and change the translucency of our pigmentation, and adapt our eyes to see in dark conditions, \
+	The translucent effect works best in dark enviroments. Can be toggled on and off."
 	button_icon_state = "chameleon_skin"
 	dna_cost = 2
 	chemical_cost = 25
 	req_human = TRUE
+	/////is ability active?
+	var/is_active = FALSE
 
 /datum/action/changeling/chameleon_skin/sting_action(mob/user)
-	var/mob/living/carbon/human/H = user //SHOULD always be human, because req_human = TRUE
-	if(!istype(H)) // req_human could be done in can_sting stuff.
-		return
+	var/mob/living/carbon/human/cling = user //SHOULD always be human, because req_human = TRUE
 	..()
-	if(H.dna.get_mutation(/datum/mutation/human/chameleon))
-		H.dna.remove_mutation(/datum/mutation/human/chameleon)
+	if(is_active == FALSE)
+		animate(cling, alpha = 85,time = 3 SECONDS)
+		cling.visible_message("<span class='warning'>[cling.name] skin is suddenly slowly becoming translucent!</span>", \
+						"<span class='notice'>You are now far more stealthy and better at looking in the dark.</span>")
+		animate(cling, color = COLOR_GRAY, time = 3 SECONDS) //darkens their overall appearance
+		var/obj/item/organ/internal/eyes/eyes = cling.get_organ_by_type(/obj/item/organ/internal/eyes)
+		eyes.lighting_cutoff = LIGHTING_CUTOFF_MEDIUM // Adds barely usable, kinda shit night vision
+		eyes.flash_protect = max(eyes.flash_protect - 1, FLASH_PROTECTION_HYPER_SENSITIVE) //Reduces flash protection by one level.
```

Comment:
This has a funky order-of-operations bug with Augmented Eyesight that needs to be fixed. 

If activate this ability, and then de-activate augmented eyesight, the flash protection it offers will completely override this.

Rather than having `FLASH_PROTECTION_WELDER - 1` protection as you would expect, it would just have `FLASH_PROTECTION_WELDER`.  

Likewise, If you had this ability active and then activated augmented eyesight, you would have `FLASH_PROTECTION_SENSITIVE` eyes rather than `FLASH_PROTECTION_HYPER_SENSITIVE`.

Reply:
I'll just change how augmented eyesight changes flash_protection to be in line withe how this new power works. is that okay?

Reply:
nevermind that just exacerbate the problem

Reply:
Probably fine, though a nerf to moth lings, which is based I guess

Reply:
Also you need to register signals for organ gained and organ lost so the eye resets its flash protection when eyes are removed and lost, like augmented eyesight.

Reply:
> Probably fine, though a nerf to moth lings, which is based I guess

sounds like a skill issue tbh